gluoncv/model_zoo/resnet.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, channels, stride, downsample=False, in_channels=0,
             self.se = nn.HybridSequential(prefix='')
             self.se.add(nn.Dense(channels // 4, use_bias=False))
```

Comment:
it should be `channels // 16` as well, according to the original paper.

Reply:
Yes, you are right. I just changed it, thank you.